.github/ISSUE_TEMPLATE/new-dbt-model.md

```diff
@@ -109,35 +119,63 @@ join {{ ref('default.vw_pin_universe') }}
 using (pin10, year)
 ```
 
+```python
+# Python model example
+# dbt/models/default/default.new_model.py
+import pandas as pd
+
+def model(dbt, spark_session):
+    raw_foobar = dbt.source("raw", "foobar")
+    vw_pin_universe = dbt.ref("default.vw_pin_universe")
+    result = pd.merge(raw_foobar, vw_pin_universe, on=["pin10", "year"])
+    dbt.write(result[["pin10", "year"]])
+```
+
 - [ ] Optionally configure model materialization. If the output of the query
   should be a view, no action is necessary, since the default for all models in
   this repository is to materialize as views; but if the output should be a
   table, with table data stored in S3, then you'll need to add a config block
-  to the top of the view to configure materialization.
+  to the top of the view to configure materialization. Note that all Python
+  models must be materialized as tables.
 
-```sql
--- Table example
--- dbt/models/default/default.new_model.sql
-{{
-  config(
-    materialized='table',
-    partitioned_by=['year'],
-    bucketed_by=['pin10'],
-    bucket_count=1
-  )
-}}
+```diff
+# Table example
+--- dbt/models/default/default.new_model.sql
++++ dbt/models/default/default.new_model.sql
++ {{
++   config(
++     materialized='table',
++     partitioned_by=['year'],
++     bucketed_by=['pin10'],
++     bucket_count=1
++   )
++ }}
```

Comment:
Changing this to a diff to make it clearer to the reader which portion of the code has been added as part of this step.